tests/Doctrine/Tests/Common/Collections/Expr/CompositeExpressionTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Doctrine\Tests\Common\Collections\Expr;
+
+use Doctrine\Common\Collections\Expr\CompositeExpression;
+use Doctrine\Common\Collections\Expr\ExpressionVisitor;
+use Doctrine\Common\Collections\Expr\Value;
+use PHPUnit_Framework_TestCase as TestCase;
+
+/**
+ * Class CompositeExpressionTest
+ *
+ * PHP Version 5
+ *
+ * @category  PHP
+ * @package   Doctrine\Tests\Common\Collections\Expr
```

Comment:
this project does not use ``@category`` and ``@package`` (duplicating the namespace in ``@package`` brings absolutely no value)